adagrams/game.py

```diff
@@ -1,11 +1,144 @@
+import random
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+
 def draw_letters():
-    pass
+    tiles = [] #list that's ten letters
+    count = 0 #going to stop at 10
+    letters_dict = LETTER_POOL.copy()
+    while count != 10:
+        letter = random.choice(list(letters_dict.keys()))
+        if letters_dict[letter] > 0:
+            letters_dict[letter] -= 1
+            tiles.append(letter)
+            count += 1
+        else:
+            continue
+    return tiles
+        
 
 def uses_available_letters(word, letter_bank):
-    pass
+    letter_bank_dict = {}
+    word_dict = {}
+    word = word.upper()
+
+    for letter in letter_bank:
+        if letter not in letter_bank_dict:
+            letter_bank_dict[letter] = 1
+        else:
+            letter_bank_dict[letter] += 1
 
+    for character in word:
```

Comment:
You can use the variable name `letter` in this for loop too instead of `character` if you'd like. While this for loop shares the same local scope as the for loop on line 52, the first loop finished running and using `letter` again works too.